samples/python/console/speech_sample.py

```diff
@@ -37,6 +37,7 @@
 weatherfilenamemp3 = "whatstheweatherlike.mp3"
 weatherfilenamemulaw = "whatstheweatherlike-mulaw.wav"
 seasonsfilename = "pronunciation_assessment_fall.wav"
+zhcnfilename = "zhcn_sample.wav"
```

Comment:
As we synced before, this file should be a short audio sample.

1. Can you rename it as `zhcn_short_dummy_sample.wav`?
2. Can you check in this wave file?

Reply:
updated.